lib/Feature/FeatureModelTransaction.cpp

```diff
@@ -7,10 +7,15 @@ namespace vara::feature {
 bool mergeSubtree(FeatureModelCopyTransaction &Trans, FeatureModel const &FM,
                   Feature &F, bool Strict);
 
+bool mergeSubtree(FeatureModelCopyTransaction &Trans, FeatureModel const &FM,
+                  Relationship &R, bool Strict);
```

Comment:
I'm not 100% sure what this method does, but maybe the name is just confusing.
Does this merge feature subtree that are related to the Relationship or does it merge relationships?

Reply:
It's unused so I removed it. Originally, planned to overload the function to handle Relationships differently during merging.

Reply:
So, if you don't need that anymore a updated name should suffice